SS_selex.tpl

```diff
@@ -1740,8 +1740,8 @@ FUNCTION void get_selectivity()
                 if (k >= 0)
                 {
                   k++;
-                  if (docheckup == 1)
-                    echoinput << a << "  sigmasel: " << sigmasel << endl;
+//                  if (docheckup == 1)
+                  temp = sel_a(y, fs, 1, a);

```

Comment:
I don't understand the impact of this `temp` variable

Reply:
Oops.  That's a leftover from my testing.  I'll clean it out.